src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -753,11 +754,12 @@ public Counters run() throws IOException {
         config = new IndexWriterConfig(germanAnalyzer);
       } else if (args.language.equals("es")) {
         config = new IndexWriterConfig(spanishAnalyzer);
-      } else if (args.language.equals("en_ws")) {
+      } else if (args.language.equals("en_ws") || args.pretokenized) {
```

Comment:
hrm... not sure what `en_ws` is... I don't think it's a valid ISO code. Let's kill?